Unit tests for reserve.py and input_util.py due to unpinned pandas dependency

## 🐛 Describe the bug
Two unit tests fail.  This is (probably) the result of a new `pandas` release, requiring a later version of `openpyxl` than the pinned `openpyxl` version.

## 📜 To Reproduce
Steps to reproduce the behavior:

Run the following or delete `./.tox` in an existing repo and run `tox`.  This bug will not present unless the tox environment is rebuilt.

```bash
cd /tmp
git clone https://github.com/NASA-PDS/doi-service.git
cd doi-service
python3 -m venv venv
source venv/bin/activate
pip install --editable '.[dev]'
# cp path/to/populated/pds_doi_service.ini ./
export CI=true; tox  # CI=true suppresses some tests which will fail without some specific, unrelated config dependencies populated
```

## 🕵️ Expected behavior
Unit tests pass

## 🩺 Test Data / Additional context

Failures are as follows

```
=========================== short test summary info ============================
FAILED src/pds_doi_service/core/actions/test/reserve_test.py::ReserveActionTestCase::test_reserve_xlsx_and_submit
FAILED src/pds_doi_service/core/input/test/input_util_test.py::InputUtilTestCase::test_read_xls
=========== 2 failed, 83 passed, 15 skipped, 145 warnings in 11.85s ============
```

resulting from 

```
E                   ImportError: Pandas requires version '3.0.7' or newer of 'openpyxl' (version '3.0.3' currently installed).
```

--- a/README.md
+++ b/README.md
@@ -140,6 +140,8 @@ Testing details are detailed in this section.
 
 ### Tox (for developers)
 
+#### N.B. Updates to pip dependencies are not automatically applied to existing tox virtual environments, to keep unit testing fast.  The simplest way to propagate dependency updates is to delete `./.tox` and run tox again.
+
 [tox](https://tox.readthedocs.io/) is installed automatically during `pip install --editable .[dev]`, and provides virtual environments and run configurations for
 - unit/functional testing
 - linting

--- a/setup.cfg
+++ b/setup.cfg
@@ -41,8 +41,8 @@ install_requires =
     nltk==3.5
     numpy>=1.18
     openapi-schema-validator==0.1.4
-    openpyxl==3.0.3
-    pandas>=1.0
+    openpyxl~=3.0
+    pandas~=1.0
     python-dateutil>=2.8
     python-jose[cryptography]
     pytz==2020.1

--- a/setup.cfg
+++ b/setup.cfg
@@ -42,7 +42,7 @@ install_requires =
     numpy>=1.18
     openapi-schema-validator==0.1.4
     openpyxl~=3.0
-    pandas~=1.0
+    pandas~=1.4
     python-dateutil>=2.8
     python-jose[cryptography]
     pytz==2020.1

